modules/new_serverpod_auth/serverpod_auth_api_tokens/serverpod_auth_api_tokens_server/lib/src/models/api_token.spy.yaml

```diff
@@ -0,0 +1,49 @@
+class: ApiToken
+serverOnly: true
+table: serverpod_auth_session
+fields:
```

Comment:
Maybe this also needs something like `extraClaims` (also exposed on the `AuthenticationInfo`), so one can easily add extra checks (e.g. what a personal access token should be able to do, if it's e.g. just for a specific resource, etc.)